uber/automated_emails.py

```diff
@@ -427,6 +427,14 @@ def __init__(self, subject, template, filter, ident, **kwargs):
     when=days_before(1, c.FINAL_EMAIL_DEADLINE),
     ident='volunteer_shift_schedule')
 
+if c.VOLUNTEER_AGREEMENT_ENABLED:
+    StopsEmailFixture(
+        'Reminder: Please agree to terms of {EVENT_NAME} ({EVENT_DATE}) volunteer agreement',
+        'signups/volunteer_agreement.txt',
+        lambda a: c.SHIFTS_CREATED and c.VOLUNTEER_AGREEMENT_ENABLED and not a.agreed_to_volunteer_agreement,
```

Comment:
The check for `VOLUNTEER_AGREEMENT_ENABLED` is redundant with the prior if statement. Not a big deal though.